app/react/V2/Components/UI/Button.tsx

```diff
@@ -2,7 +2,7 @@ import React, { MouseEventHandler } from 'react';
 
 interface ButtonProps {
   children: string | React.ReactNode;
-  styling?: 'solid' | 'outline' | 'light';
+  styling?: 'solid' | 'outline' | 'light' | 'action';
```

Comment:
"action" is very similar to "light", exepct it has no hover. Maybe we don't need a fourth type of button, just modify light to be more like action?

Reply:
I did not want to modify light because it is being used by other parts of the UI